src/types/types.tsx

```diff
@@ -0,0 +1,14 @@
+export type User = {
```

Comment:
Now the name `User` and `Todo` is well understood in the types file itself, but when we reuse this types, it will be more convenient to give a name that will directly inform that it is a types, interfaces, enams, and so on. (For example: `IUser/ITodo`, `UserInterface/TodoInterface`, `TUser/TTodo`, `UserType/TodoType`).